web/usuaris/src/components/organisms/FiltersWidget.tsx

```diff
@@ -21,6 +23,15 @@ export const FiltersWidget: FC<TFiltersWidget> = ({ filters, setFilters }) => {
   const handleItinerary = (itinerary: TItinerary | undefined) => {
     setFilters({ ...filters, itinerarySlug: itinerary?.slug })
   }
+  const handleRole = (role: TRol | undefined) => {
```

Comment:
With the changes done in the qs you don't need to write the conditional, you should just put ```setFilters({ ...filters, role: role.slug })``` like in the handleItinerary